src/pages/addons.tsx

```diff
@@ -1,15 +1,350 @@
 import { pricingMenu } from '../navs'
 import Layout from 'components/Layout'
-import { SectionHeader } from 'components/Pricing/Test/Sections'
-import Link from 'components/Link'
-import Tooltip from 'components/Tooltip'
-import { IconInfo } from '@posthog/icons'
-import { CTA as PlanCTA } from 'components/Pricing/Plans'
+import { IconAdvanced } from '@posthog/icons'
+import { usePlatform } from 'components/Pricing/Platform/usePlatform'
+import useProducts from 'components/Pricing/Products'
+import { PricingTiers } from 'components/Pricing/Plans'
+import { Link } from 'react-scroll'
 import { StaticImage } from 'gatsby-plugin-image'
+import { CallToAction } from 'components/CallToAction'
+
+const FeatureItem = ({ icon: Icon, name, description, size }) => (
+    <div className="flex gap-2">
+        <div className="shrink-0 hidden">
+            <Icon className="size-8" />
+        </div>
+        <div className="flex-1">
+            <h4 className="text-base my-1">{name}</h4>
+            <p
+                className={`${size === 'small' ? 'text-sm [&_li]:text-sm' : 'text-[15px] [&_li]:text-[15px]'} text-primary/75 dark:text-primary-dark/75 mb-0`}
+                dangerouslySetInnerHTML={{ __html: description }}
+            />
+        </div>
+    </div>
+)
+
+const addons = [
+    {
+        name: 'Teams',
+        features: [
+            {
+                group: 'Features',
+                items: [
+                    {
+                        name: 'Unlimited projects',
+                        description:
+                            'Create silos of data within PostHog. All data belongs to a single project and all queries are project-specific.',
+                        icon: IconAdvanced,
+                    },
+                    {
+                        name: 'Managed reverse proxy',
+                        description:
+                            'Use our managed proxy service to send events through your own domain. Capture more usage data that might otherwise be intercepted by ad blockers without needing to set up and maintain the proxy yourself.',
+                        icon: IconAdvanced,
+                    },
+                    {
+                        name: 'White labeling',
+                        description: 'Use your own branding on surveys, shared dashboards, shared insights, and more.',
+                        icon: IconAdvanced,
+                    },
+                    {
+                        name: 'Ingestion taxonomy',
+                        description:
+                            'Mark events as verified or unverified to help you understand the quality of your data.',
+                        icon: IconAdvanced,
+                    },
+                    {
+                        name: 'Dashboard tags',
+                        description: 'Organize dashboards with tags.',
+                        icon: IconAdvanced,
+                    },
+                ],
```

Comment:
Sure thing, if you can add these in (in these groupings), awesome. Otherwise if you can't get to it soon, let's merge and we can revisit?

Reply:
cc @smallbrownbike

Reply:
Ok, we're now snagging Teams features from the API! The others are still using @corywatilo's custom descriptions (it looks like the API only returns the basic description for those at the moment)

One other thing - **Project permissions**, **Ingestion taxonomy**, and **Dashboard tag**s weren't assigned a category in the API, so they're showing up under "Features" right now. Not sure if this was intentional but calling it out in case it wasn't!

Reply:
Can we get those categorized @raquelmsmith?

Reply:
This is how they were categorized in the original mock:

![image](https://github.com/user-attachments/assets/25b1c1c3-3b80-43fb-998c-e16c67fb02b3)

LMK if yo uwant them categorized elsewhere though. They seem fine to me to be called Features.